hasura/graphql-engine
# [GQL-19] #8537 Move the scroll/count of rows facility to a position that is above the table, not below as currently

Description
The issue:
The pagination of the table on browse row tab was below the table due to which the user have to scroll all the way down to see the pagination in case of 100's of rows. And even the header of the table was not visible.

### Suggestion
Move the scroll/count of rows facility to a position that is above the table, not below as currently. 
So this PR tries to solve the issue mention above by shifting the pagination of the table above it and making the table header sticky. 

### Changelog


### Affected components



### Related Issues
[#hasura-8537](https://github.com/hasura/graphql-engine/issues/8537)

### Solution and Design
### Issue
https://user-images.githubusercontent.com/81171209/176469614-31cdbbc2-0736-4a62-ad07-767407027544.mp4

### Solution

https://user-images.githubusercontent.com/81171209/176641701-96462f4b-892d-457e-aeae-59b7fd06a921.mp4

### Steps to test and verify

- Navigate to the Data tab
- In a browse row you can see inserted data.(If not inserted insert the data from insert tab)
- You can see the pagination above the table.
- Scroll the page to see sticky table header.

### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  2. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  3. Logging
